spark/src/main/java/org/apache/iceberg/Actions.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg;
+
+import org.apache.spark.sql.SparkSession;
+
+public class Actions {
```

Comment:
This API is useful and extendable for other usecases. Would it be a good idea to add this to the Iceberg Spec? (In a separate followup issue maybe)

Reply:
I think it's good to have this API (which was actually added in #873), but I wouldn't make it part of the spec. The spec covers at-rest data tracking, not how the implementations work.

That said, I think it would be great to add some documentation for this on the site! Both for people to know what actions will be available in 0.8.0 and to document how to build actions to include in Iceberg. I'd like to get our data warehouse team to build their planned actions this way so we can easily share them.

Reply:
Yes, we would definitely want to document that before the release. I did not do that in the first place as we still might change some things.